build.cmd

```diff
@@ -99,11 +99,15 @@ set __PassThroughArgs=%__PassThroughArgs% %1
 
 if /i "%1" == "clean"               (set __CleanBuild=1&shift&goto Arg_Loop)
 
+REM CI_TODO: Remove this once CI stops building mscorlib for non-Windows OS on Windows.
+REM Need to keep FreeBSD until .NET CLI Exists for it.
```

Comment:
I would really like to keep the ability to cross compile mscorlib for other OS's on Windows.  We support this in CoreFX for the rest of the managed code and it's very useful for both bootstrapping and local development.